### ISSUE TITLE
TransparentNode bounding boxes should be clamped to the size of the visible object

### ISSUE THREAD

**Issue by bloxgate** [Electrical-Age/ElectricalAge#792]
The bounding box for a covered DC-DC converter is just slightly larger than the actual model, making it impossible to edge around one if its on a ledge.

[![https://gyazo.com/3d9b6dcadb2d8271426a2c5eb512e29d](https://i.gyazo.com/3d9b6dcadb2d8271426a2c5eb512e29d.png)](https://gyazo.com/3d9b6dcadb2d8271426a2c5eb512e29d)

**Comment by AMIDIBOSS** [comment]
Eh, Yes it is...
That's called transparent node. Every transparent node has bounding box 1x1x1

**Comment by bloxgate** [comment]
Just because its a transparent node doesn't mean it has to be 1x1x1 though. I'd like it if it matched the model a bit more in this case.

**Comment by Baughn** [comment]
Yes, that would indeed be nice, and could be done automatically. In fact, we already have 90% of the code needed; what's left is a plumbing.

I generalized the title of the issue a little. ;-)
